SparkPost/2web2ui
# SA-1852 - (UI) Compare By - Rejection Reasons Report

[SA-1852](https://sparkpost.atlassian.net/browse/SA-1852)

### What Changed
- Adds new rejection reason tabs when the user has relevant metrics selected as well as active comparisons

### How To Test
1. Go to `/signals/analytics` with the compare by account UI option enabled
2. Add two active comparisons
3. Add rejection-related metrics - "Rejected", "Policy Rejections", "Generation Failures", "Generation Rejections", and/or "Rejection Rate"
4. Verify that a new rejection reason tab with a table renders for each active comparison
5. Verify that each tab makes network requests for rejection reasons by domain with the relevant comparison passed in as a filter in the query param
6. Verify tests properly test changes

### To Do
